pontoon/base/static/js/translate.js

```diff
@@ -429,7 +429,21 @@ var Pontoon = (function (my) {
      * Update current translation length
      */
     updateCurrentTranslationLength: function () {
-      $('#translation-length .current-length').html($('#translation').val().length);
+      var limit = this.translationLengthLimit,
+          translation = $('#translation').val();
+
+      if (limit) {
+        var length = this.stripHTML(translation).length,
```

Comment:
Haha, a totally good point :-)
I just thought that MAX_LENGTH is a protection mechanism against strings that won't fit in the viewport or something like that.